solvers/DPStokes/mobility.h

```diff
@@ -44,30 +44,110 @@ class DPStokes: public libmobility::Mobility{
     dpstokes = std::make_shared<uammd_dpstokes::DPStokesGlue>();
   }
 
+  double configure_grid_and_kernels_xy(Parameters ipar){
+    double hydroRadius = ipar.hydrodynamicRadius[0];
+
+    // STEP 1: set w & beta based on kernel type (Table 1/2)
+    double w = 6.0; // note: should be an int but I'm scared of the consequences
+    double beta = 1.714;
```

Comment:
I thought this effort was in order to avoid using this magic number?

Reply:
I thought so too, but it seems somewhat nuanced. The kernel parameters determine the effective hydrodynamic radius $R$, but this radius will have slight numerical variation depending on exactly where the particle is with respect to the grid. From reading the paper and looking at their code, it seems these values are the optimal values for the kernel that best preserve translational invariance of $R$ while a particle is moving across a discrete grid. However, for a domain of length $L$ and a grid spacing $h$, they also want the number of points $N_x = L/h$ to be an FFT-friendly number. Not keeping $\beta$ at the exactly optimal values seems like it gives you some wiggle room to find a better grid spacing and therefore a better number of grid points. 

The choice of $w$ might be able to be selected from a tolerance or whether torques are needed. $w=4$ can be used and preserves grid invariance only for forces, but fails when torques are applied. $w=5,6$ works for scenarios where both forces and torques are needed. I'll have to read a bit more to see if I can figure out how to relate $w$ to a tolerance.

Reply:
I believe leaving w=6 is ok, the optimal value of beta changes for different w. Its ok what you did should solve our immediate issue with accuracy.